config/config.go

```diff
@@ -72,6 +72,11 @@ func (c Config) Validate() error {
 				return fmt.Errorf("Missing room in HipChat config: %s", proto.MarshalTextString(hcc))
 			}
 		}
+		for _, sc := range nc.SlackConfig {
+			if sc.WebhookUrl == nil {
+				return fmt.Errorf("Missing webhook url in Slack config: %s", proto.MarshalTextString(sc))
```

Comment:
In the same style, should I rename `contentTypeJson`, `pagerdutyApiUrl` and `hipchatUrl` ?